AzureADSample/AzureADSample/ViewModel/ProfilePageViewModel.cs

```diff
@@ -8,4 +10,16 @@ public partial class ProfilePageViewModel : ObservableObject
 {
     [ObservableProperty]
     User user;
+    async void LogOut(object sender, EventArgs e)
+    {
+        IEnumerable<IAccount> accounts = await AuthService.authenticationClient.GetAccountsAsync();
+
+        while (accounts.Any())
+        {
+            await AuthService.authenticationClient.RemoveAsync(accounts.First());
+            accounts = await AuthService.authenticationClient.GetAccountsAsync();
```

Comment:
Add `SecureStorage.Default.RemoveAll();` Because I've initially also store the token securely on the user's device so we must also remove it once logged out.